src/NasaImg.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import styled from 'styled-components'
+function NasaImg(props){
+    
+    console.log(props)
+    
+    const ImgSrc = styled.img`
+    
+    width:50%;
+    border: 3px solid white;
+    margin-bottom: 2.5rem;
+    `
+
+    return(
+    <div>
+        <ImgSrc alt = 'Image of space' src = {props.img}/>
```

Comment:
A suggestion would be instead of hardcoding the `alt` to be `Image of space` since this will change over time.
I suggest pulling this data from the actual `props` and the api call so it is dynamic and synced with the image.